card/src/main/java/com/adyen/checkout/card/CardViewProvider.kt

```diff
@@ -24,6 +27,8 @@ internal object CardViewProvider : ViewProvider {
     ): ComponentView {
         return when (viewType) {
             CardComponentViewType -> CardView(context, attrs, defStyleAttr)
+            Adyen3DS2ComponentViewType -> PaymentInProgressView(context, attrs, defStyleAttr)
```

Comment:
Hmm we shouldn't really do that, we'll need to add the other action views and do the same for all the view providers. But I don't have an alternative yet, let's think about it

Reply:
Oh yeah I wanted to discuss this, because this feels wrong indeed. Couldn't think of anything better for now